README.md

```diff
@@ -18,7 +18,7 @@ This will install all runtime dependencies, as well as Webpack build and Mocha/C
 
 ### Environment variables
 
-We are using the `dotenv` package and `.env` file to emulate environment variables in development mode. Your `.env` file should have the following:
+We are using the `dotenv` package and `.env` file to emulate environment variables in development mode. You should create a `.env` file at the root and it should have the following:
 
 ```
 SESSION_SECRET=YOUR_SECRET
```

Comment:
We should take '.env' out of the .gitignore while we're testing. General comment, not specific to your commit.